04_containers.Rmd

```diff
@@ -725,13 +725,13 @@ sample_meta[,1] <- NULL
 colnames(sample_meta) <- c("patient_status", "cohort", "patient_status_vs_cohort", "sample_name")
 
 # Then it can be added to colData
-colData(se) <- sample_meta
+se <- sample_meta
```

Comment:
Should be removed. The colData(se) is already correctly assigned in the previous version. In the new version you are replacing the entire SE object with the column data. But the column data belongs to `colData(se)`, not in `se`.